addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -39,10 +39,9 @@
                     <h3 t-attf-class="col-lg-10 mb-0" t-out="question.name"/>
                     <div class="col d-flex justify-content-end align-items-center">
                         <i t-if="question.state == 'close'" class="fa fa-lock ms-2 fs-4" title="Closed" data-bs-toggle="tooltip" data-bs-placement="top"/>
-                        <span t-elif="not question.active" class="badge bg-danger">
-                            <t t-if="question.state!='offensive'">Deleted</t>
-                            <t t-if="question.state=='offensive'">Offensive</t>
-                            <t t-if="question.state=='offensive' and question.closed_reason_id">
+                        <span t-elif="not question.active" class="badge bg-danger">Deleted</span>
+                        <span t-elif="question.state == 'offensive'" class="badge bg-danger">Offensive
```

Comment:
don't we need a margin between the two here? Just guessing at this point

Reply:
they will not show together ever because of t-elif, but if we decide that we want to show both badges at the same time in case of deleted offensive posts(the reason why I don't think it's necessary is on comment below), then yes, we will need a few pixel margin on left :D